docs/source/new-features/index.rst

```diff
@@ -46,6 +67,82 @@ Find out about the details of major features, changes, and deprecations below.
                     </div>
                 </div>
             </div>
+            <div class="grid-card">
+                <div class="content-container">
+                    <div class="header">
+                        Overhaul of MLflow Tracking UI for Deep Learning workflows
+                    </div>
+                    <img class="card-image" src="../_static/images/logos/mlflow-logo.svg" alt="MLflow"></img>
+                    <div class="body">
+                        We've listened to your feedback and have put in a huge amount of new UI features designed to empower and 
+                        simplify the process of evaluating DL model training runs. Be sure to upgrade your tracking server and 
+                        benefit from all of the new UI enhancements today!
+                    </div>
+                    <div class="tag">
+                        <a href="https://github.com/mlflow/mlflow/releases/tag/v2.11.0">released in 2.11.0</a>
+                    </div>
+                </div>
+            </div>
+            <div class="grid-card">
+                <div class="content-container">
+                    <div class="header">
+                        Automated model checkpointing support added for Deep Learning model training
+                    </div>
+                    <img class="card-image" src="../_static/images/logos/mlflow-logo.svg" alt="MLflow"></img>
+                    <div class="body">
+                        When performing training of Deep Learning models with PyTorch Lightning or Keras, model checkpoint saving 
+                        is enabled, allowing for state storage during long-running training events and the ability to resume if 
```

Comment:
nit: Do we have a doc reference for how to enabling and configuring the checkpointing?

Reply:
Good thing you pointed this out - we were missing the docstrings for tensorflow! Added those and updated with links to the API docs